Dockerize django app

mhmdahmedfathi opened the pull request:
- I was able to dockerize the Django app by making 3 images, which are Postgres, Redis, and the Django app, each running in its own specific port, and I use ports so they can call each other.
- Change the readme so it will be clear to all how they can use Docker
- add some dependencies that were missing in the app that cause the app to not be running with me

romain-fontugne commented:
thanks for your contribution!
But I'd like to not include the postgres images here. Other modules/scripts are pushing data to our postgres database so I want to keep it separate. We could keep it for testing but I'd prefer if by default it doesn't include postgres.

Why do we need redis?

mhmdahmedfathi commented:
You are welcome.
Okay, I will change it and include it in the same commit, and I will let you know.
You can find redis included in const.py, and we use redis to make channels so we can send emails, socket anything we want.

mhmdahmedfathi commented:
Hello @romain-fontugne , I have added to this commit, the changes that you want and added some other which are :

- the ability to connect to the local postgres
- the steps to do local connection with postgres or use image postgres

romain-fontugne commented:
Thanks, I will try that soon

mhmdahmedfathi commented:
You are welcome. Let me know if there is anything that is ambiguous so I can try to explain or help if possible.

romain-fontugne commented:
Thanks very much!
